docs/API/REST-V2-Usage.md

```diff
@@ -695,7 +695,7 @@ A patch is a manually initiated version submitted to test local changes.
 | git_hash              | string         | Hash of commit off which the patch was initiated                                                                                     |
 | patch_number          | int            | Incrementing counter of user's patches                                                                                               |
 | author                | string         | Author of the patch                                                                                                                  |
-| status                | string         | Status of patch                                                                                                                      |
+| status                | string         | Status of patch (created, started, success, failed)                                                                                  |
```

Comment:
I just documented a bunch of these in #6580, so maybe remove so we can avoid merge conflicts?